tools/ios-eddystone-scanner-sample/EddystoneScannerSample/ESSEddystone.m

```diff
@@ -75,8 +76,7 @@ - (NSString *)description {
   if (self.beaconType == kESSBeaconTypeEddystone) {
     return [NSString stringWithFormat:@"ESSBeaconID: beaconID=%@", self.beaconID];
   } else {
-    return [NSString stringWithFormat:@"ESSBeaconID with invalid type %lu",
-        (unsigned long)self.beaconType];
+    return [NSString stringWithFormat:@"ESSBeaconID with invalid type %lu", (unsigned long)self.beaconType];
```

Comment:
Which is correct?

```
return [NSString stringWithFormat:@"ESSBeaconID with invalid type %lu",
    (unsigned long)self.beaconType];
```

or

```
return [NSString stringWithFormat:@"ESSBeaconID with invalid type %lu",
  (unsigned long)self.beaconType];
```

I couldn't judge I should use 2 space or 4 space for this case.